### PR TITLE
Add more info about channels page

### PR DESCRIPTION
**Pull request by paigecalvert**
Previews: 
- Promoting [and Editing] Releases: https://deploy-preview-795--replicated-docs.netlify.app/vendor/releases-promoting
- About Release Channels: https://deploy-preview-795--replicated-docs.netlify.app/vendor/releases-about-channels

### CODE REVIEW

File: docs/reference/replicated-cli-release-promote.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ replicated release promote SEQUENCE CHANNEL_ID [Flags]
   <tr>
     <td><code>--required</code></td>
     <td></td>
-    <td>Prevents users from skipping this release during application upgrades. For more information, see <a href="/vendor/releases-promoting">Promoting Releases</a>.</td>
+    <td>Prevents users from skipping this release during application upgrades. For more information, see <a href="/vendor/releases-promoting">Promoting and Editing Releases</a>.</td>
```
Comment (paigecalvert) [inline docs/reference/replicated-cli-release-promote.md:33]: I updated the title of the topic to Promoting and Editing Releases, because I wasn't sure that just "Promoting" would capture it, making the editing info hard to find.

That said, it might be misleading to say "Editing Releases", since you're just editing release properties.

Curious for your thoughts

File: docs/vendor/releases-about-channels.md [hunk 2]
```diff
@@ -1,24 +1,49 @@
...
+
+<img src="/images/channel-card.png" alt="Channel card in the vendor portal" width="300"/>
+
+[View a larger version of this image](/images/channel-card.png)
+
+As shown in the image above, you can do the following to manage a channel from the Channels page:
+
+* Edit the channel settings by clicking on the settings icon, or archive the channel by clicking on the trash can icon. See [Creating and Editing Channels](releases-creating-channels).
+
+* In the **Adoption rate** section, view data on the adoption rate of releases promoted to the channel among customers assigned to the channel.
```
Comment (paigecalvert) [inline docs/vendor/releases-about-channels.md:35]: Curious for your thoughts/suggestions on the formatting of this section. Ended up going with a bulleted list rather than subheadings since the info is relatively light for each item. 

I also bolded the section names, which doesn't align with the style guide, but it seemed hard to parse without the bold. 

Any other ideas or suggestions?

File: docs/vendor/releases-creating-channels.md [hunk 3]
```diff
@@ -68,10 +68,6 @@ To archive a channel with the vendor portal or the replicated CLI:
 
      For more information, see [channel delete](/reference/replicated-cli-channel-delete) in the replicated CLI documentation.
 
-## Next Steps
-
-[Creating a release](releases-creating-releases)
-
```
Comment (paigecalvert) [inline docs/vendor/releases-creating-channels.md:74]: In revisiting this, it seemed odd to suggest a next step is creating a release. They probably have already created some releases before messing with channels. And these topics in the Vendor section now seem outside the ordered "Distributing Workflow". They seem more meant to serve as discrete tasks that you can do at any stage.

What do you think?

File: docs/vendor/releases-creating-releases.md [hunk 4]
```diff
@@ -6,8 +6,6 @@ Alternatively, you can use the replicated CLI and API to automate releases. For
 
 Replicated lets you add custom resources to your releases, which are packaged as part of the application but not deployed to the cluster. When included, custom resources are consumed by the app manager, the admin console, or by other kubectl plugins to control the application experience. For more information about the custom resources, see [About Custom Resources](../reference/custom-resource-about).
 
-We recommend that you bookmark the vendor portal because it is used to manage and deploy application releases.
-
```
Comment (paigecalvert) [inline docs/vendor/releases-creating-releases.md:10]: This is unrelated, but this just seemed like a silly note to be randomly appearing here

File: docs/vendor/releases-promoting.md [hunk 5]
```diff
@@ -41,12 +38,22 @@ To promote a release:
...
+
+To edit release properties:
+
+1. Log in to the vendor portal and click **Channels**.
+1. On the Channels page, in the Latest release section, click **Release history**.
 
-## Next Steps
+   The Release History page lists all releases previously promoted to the channel.
 
-[Creating customers and downloading licenses](releases-creating-customer)
```
Comment (paigecalvert) [inline docs/vendor/releases-promoting.md:47]: Similar note as above, where it didn't feel as logical to prescribe an order to these topics now that we have the Distributing workflow. Removed the next steps section and moved the link to related topics

**Comment by paigecalvert** [comment]
@jonquil2002, this is ready for an initial review. I added some context to my edits in the comments of this PR and also called out some specific areas where I'd love your thoughts in particular. LMK if it's easier to provide feedback through pairing

File: docs/vendor/releases-creating-channels.md (hunk 3 above)
Comment (jonquil2002) [reply docs/vendor/releases-creating-channels.md:74]: Agreed, the Next Step can be removed because we have that workflow topic and better tutorials. Before, it was trying to serve two purposes and now can just exist as a discreet task.

File: docs/vendor/releases-creating-releases.md (hunk 4 above)
Comment (jonquil2002) [reply docs/vendor/releases-creating-releases.md:10]: Agreed, it's a carryover that we inherited but isn't necessary.

File: docs/vendor/releases-promoting.md (hunk 5 above)
Comment (jonquil2002) [reply docs/vendor/releases-promoting.md:47]: Agreed.

File: docs/reference/replicated-cli-release-promote.md (hunk 1 above)
Comment (jonquil2002) [reply docs/reference/replicated-cli-release-promote.md:33]: I'd still like to see Creating/Promoting Releases combined and Editing Release Properties could be a separate, discreet topic.

Wondering if there should be an About Releases topic for all of that conceptual info, then a Creating and Promoting Releases topic, and then Editing Release Properties?

I think it's good if these stay as discreet topics since they're doing that linking and it might be helpful to have these discreet topics be able to function as help topics. Open to your thoughts/pairing though.

Comment (paigecalvert) [reply docs/reference/replicated-cli-release-promote.md:33]: Yeah, I think the idea of pulling out the conceptual info makes sense. I still struggle with the fact that we have all these topics for pretty basic steps in the vendor portal UI, when it's the conceptual info that's the real part that can be more challenging to understand.

Maybe there's a conceptual topic to describe the idea of creating and promoting releases, then another topic for "Managing Releases" or something generic like that that can house all the little procedures for creating/promoting/editing releases in the vendor portal.

Comment (paigecalvert) [reply docs/reference/replicated-cli-release-promote.md:33]: Though this starts to bleed into the question of creating a release versus packaging and application and what's the difference

Comment (paigecalvert) [reply docs/reference/replicated-cli-release-promote.md:33]: Okay, processing....I think I am going to scratch these changes all together for this story and just get out the updates to the About Release Channels topic. There might be too much to rework and don't want to hold up the request to add info about the Release history page.

**Comment by paigecalvert** [comment]
Closing in favor of https://github.com/replicatedhq/replicated-docs/pull/803.

Decided to narrow the scope to get out the information about the channel page. (This PR does too much to the procedures in this section, which need lots of reorg)
